### PR TITLE
backport CLI install script into GLPi 0.90.x

### PR DESCRIPTION
**Pull request by btry**
Attempt to backport cliinstall.php; see #1142

### CODE REVIEW

File: inc/dbconnection.class.php [hunk 1]
```diff
@@ -54,6 +54,32 @@ static function getTypeName($nb=0) {
 
 
    /**
+    * Create GLPI main configuration file
+    *
+    * @since 9.1
```
Comment (trasher) [inline inc/dbconnection.class.php]: Bad version ;)

File: inc/toolbox.class.php [hunk 2]
```diff
@@ -2334,6 +2334,38 @@ static function cleanNewLines($string) {
 
 
    /**
+    * Create the GLPI default schema
+    *
+    * @since 9.1
```
Comment (trasher) [inline inc/toolbox.class.php]: Bad version ;)

File: inc/dbconnection.class.php (hunk 1 above)
Comment (btry) [inline inc/dbconnection.class.php]: Yes : bad version, but actually, it has been released for GLPi 9.1 first :) 

I'll change this if the PR is considered acceptable.

**Comment by remicollet** [comment]
As "tools" folder is usually not considered as supported (not in the official archive), I don't see big problem to add this file.

BTW, better to move the 2 methods directly in cliinstall script, rather than adding them in the CommonDBTM and Toolbox classes, as they are not used by the install/update process in 9.1 (so, duplicated code)

More the 'GLPI_SYSTEM_CRON' is really 9.1 only.

So, I think it is ok to add a single file in tools, without changing anything else = no risk.
(I will be mostly against a full backport of the feature)

File: inc/dbconnection.class.php [hunk 3]
```diff
@@ -470,4 +470,4 @@ static function changeCronTaskStatus($enable=true) {
    }
 
 }
-?>
\ No newline at end of file
+?>
```
Comment (remicollet) [inline inc/dbconnection.class.php]: please, don't change, this, quash your commit, and everything will be fine for me.

Comment (remicollet) [inline inc/dbconnection.class.php]: @trasher bad closing line can raise awfull issues... (because of unwanted output, when sending doc to browser).

**Comment by remicollet** [comment]
Squashed, amended and merged.

Thanks for your patience.

**Pull request closed without merging**